libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -208,7 +208,7 @@ auto start([[maybe_unused]] irods::default_re_ctx& re_ctx, const std::string& _i
 
 	try {
 		auto time_ms = ts_clock::now().time_since_epoch() / std::chrono::milliseconds(1);
```

Comment:
should be `uint64_t` instead of `auto`

Reply:
Could replace this with std::chrono types instead of integer types too.

Reply:
`uint64_t` was a deliberate choice, as it's the underlying type we want nlohmann-json to use.